app/src/main/res/layout/fragment_splash.xml

```diff
@@ -1,7 +1,28 @@
 <?xml version="1.0" encoding="utf-8"?>
-<TextView xmlns:android="http://schemas.android.com/apk/res/android"
-    style="@style/TextAppearance.AppCompat.Headline"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:gravity="center"
-    android:text="@string/splash_fragment_title" />
\ No newline at end of file
+    android:background="@color/purple">
+
+    <ImageView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:src="@drawable/splash_background_image"
+        tools:ignore="ContentDescription" />
+
+    <ImageView
+        android:layout_width="@dimen/scooter_image_width"
+        android:layout_height="@dimen/scooter_image_height"
+        android:layout_gravity="center_vertical"
+        android:layout_marginStart="@dimen/scooter_image_margin_start"
```

Comment:
Avoid using negative margins, especially with very high (uhmm low) values. It can result in weird placements on some screen